### DISCUSSION TITLE
翻译人员信息统计 / Translator Information Statistics

### DISCUSSION THREAD

**Discussion by hiroi-sora** [hiroi-sora/Umi-OCR#449] [Announcements]
Umi-OCR 使用在线平台 [Weblate](https://hosted.weblate.org/engage/umi-ocr/) 进行本地化翻译协作。点击下方网址即可参与翻译工作，您可校对、补充现有语言，或添加新语言。

- https://hosted.weblate.org/engage/umi-ocr/

本帖记录所有参与了本地化翻译的译者信息，包括名称、个人主页链接。这些信息将会展示在项目Readme，以及软件的“关于”面板中。

如果您是译者，希望校对个人信息或补充缺漏，请在本贴中回复。感谢各位对本项目的支持！

Umi-OCR uses the online platform [Weblate](https://hosted.weblate.org/engage/umi-ocr/) for collaborative localization translation. We welcome any user to participate in the translation work; you can review, supplement existing languages, or add new languages.

This post records all the translators' information who have participated in the localization translation, including names and personal homepage links. This information will be displayed in the project's Readme, as well as the "About" panel of the software.

If you are a translator and would like to review your personal information or fill in any missing details, please reply to this post. Thank you for your support of this project!

## 译者统计表 / Translator Statistics Table

Currently, only `English, 繁體中文, 日本語, Russian, Português, தமிழ்` languages are being tallied.

The rankings in the table are in no particular order.

| translator                   | url                                                        | languages                 |
| ---------------------------- | ---------------------------------------------------------- | ------------------------- |
| bob                          | https://hosted.weblate.org/user/q021                       | English, 繁體中文, 日本語 |
| Qingzheng Gao                | https://github.com/QZGao                                   | English, 繁體中文         |
| Weng, Chia-Ling              | https://hosted.weblate.org/user/ChiaLingWeng               | English, 繁體中文         |
| linzow                       | https://hosted.weblate.org/user/linzow                     | English, 繁體中文         |
| Marcos i                     | https://hosted.weblate.org/user/ultramarkorj9              | English, Português        |
| Eric Guo                     | https://hosted.weblate.org/user/qwedc001                   | English                   |
| steven0081                   | https://hosted.weblate.org/user/steven0081                 | English                   |
| Brandon Cagle                | https://hosted.weblate.org/user/random4t4x14               | English                   |
| plum7x                       | https://hosted.weblate.org/user/plum7x                     | 繁體中文                  |
| hugoalh                      | https://hosted.weblate.org/user/hugoalh                    | 繁體中文                  |
| Anarkiisto                   | https://hosted.weblate.org/user/Anarkiisto                 | 繁體中文                  |
| ドコモ光                     | https://hosted.weblate.org/user/umren190402                | 日本語                    |
| 杨鹏                         | https://hosted.weblate.org/user/ypf                        | Português                 |
| Вячеслав Анатольевич Малышев | https://hosted.weblate.org/user/1969                       | Русский                   |
| Muhammadyusuf Kurbonov       | https://hosted.weblate.org/user/muhammadyusuf.kurbonov2002 | Русский                   |
| தமிழ்நேரம்                       | https://hosted.weblate.org/user/TamilNeram/                | தமிழ்                       |

update: `2025/03/25`

**Comment by steven0081** [comment]
很喜欢这个软件，功能真的是不错，我也想参与翻译的校对工作。请问怎么参与。

**Reply by hiroi-sora** [reply]
你好，只需要用浏览器访问下述网址就行啦~

https://hosted.weblate.org/engage/umi-ocr/

**Comment by manoloss612** [comment]
什么时候OCR的语言可以支持法语？

**Reply by Lydon-808** [reply]
同问同问，还求德语

**Comment by SmileLifeIven** [comment]
http方式接口返回内容更新了吗？与文档对不上，拿不到任务id

**Comment by zhkqqkzh** [comment]
这个能支持Windows10吗
